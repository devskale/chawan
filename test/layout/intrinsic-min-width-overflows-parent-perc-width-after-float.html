<!DOCTYPE html>
<body>
<div style="float: right">right</div>
<div style="display: flow-root; background: purple">
<div style="margin-left: auto; margin-right: auto; width: 100%; background: red">
<pre>
test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test test 
