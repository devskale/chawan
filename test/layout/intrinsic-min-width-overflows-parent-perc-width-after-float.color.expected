                                                                           right
[48;2;128;0;128m                                                                                [49m
[48;2;255;0;0mtest test test test test test test test test test test test test test test test [49mtest test test test test test test test test test test test test test test 
[48;2;128;0;128m                                                                                [49m
